udata/core/dataset/rdf.py

```diff
@@ -458,6 +458,7 @@ def resource_from_rdf(graph_or_distrib, dataset=None, is_additionnal=False):
     resource.filesize = rdf_value(distrib, DCAT.byteSize)
     resource.mime = mime_from_rdf(distrib)
     resource.format = format_from_rdf(distrib)
+    resource.schema = schema_from_rdf(distrib)
```

Comment:
Since schema is not properly exposed in most catalogs, it would be nice to assign it only if found, to prevent erasing what users might fill on datagouv direclty:
```suggestion
     schema = schema_from_rdf(distrib)
     if schema:
         resource.schema = schema
```